sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -461,6 +482,14 @@ def __init__(self, transform_executor, committed_bundle=None,
         # Not the right exception.
         self.exc_info = (exception, None, None)
 
+    def __repr__(self):
+      return "%s(%s, %s, %s)" % (
+          self.__class__.__name__,
+          'has commmitte_bundle' if self.committed_bundle else 'x',
+          'has unprocessed_bundle' if self.unprocessed_bundle else 'x',
+          self.exception)
```

Comment:
<!--new_thread; commit:d61a79d86a74422660f4959a8abb348f9f1d5129; resolved:0-->
We can keep this for now, but let's not forget to remove debugging output later.